tests/acceptance/join-section-button-test.js

```diff
@@ -0,0 +1,19 @@
+import { module, test } from 'qunit';
+import { click, visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'website-www/tests/helpers';
+
+module('Acceptance | join section button', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /', async function (assert) {
+    await visit('/');
+
+    assert.strictEqual(currentURL(), '/');
+
+    assert.dom('.join-section a.button').exists();
```

Comment:
We can use data selector here since it is repeated here

Reply:
@satyam73 is that only for some optimization ?